content/en/docs/ops/configuration/extensibility/wasm-pull-policy/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Pull Policy for WebAssembly Modules
+description: Describes how to determine whether newly pulling Wasm modules or using the cached one.
+weight: 10
+aliases:
+  - /help/ops/extensibility/wasm-pull-policy
+  - /docs/ops/extensibility/wasm-pull-policy
+keywords: [extensibility,Wasm,WebAssembly]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: n/a
+status: Alpha
+---
+
+[WasmPlugin API](/docs/reference/config/proxy_extensions/wasm-plugin) provides the way of [distributing Wasm module](/docs/tasks/extensibility/wasm-module-distribution) to proxies.
+Since each proxy should pull Wasm modules from remote registry or HTTP server, understanding of the pull policy would be important in terms of usability as well as performance.
```

Comment:
```suggestion
The [WasmPlugin API](/docs/reference/config/proxy_extensions/wasm-plugin) provides a method for [distributing Wasm modules](/docs/tasks/extensibility/wasm-module-distribution) to proxies.
Since each proxy will pull Wasm modules from a remote registry or an HTTP server, understanding how Istio chooses to pull modules is important in terms of usability as well as performance.
```